java-util/src/main/java/io/druid/java/util/common/concurrent/SameThreadExecutorService.java

```diff
@@ -28,6 +28,11 @@
  */
 public class SameThreadExecutorService extends AbstractExecutorService
 {
+  // Use io.druid.java.util.common.concurrent.Execs#sameThreadExecutor()
```

Comment:
Could you please make this a Javadoc comment

Reply:
Or just move to class-level comment

Reply:
Added class level comment